omnigibson/prims/entity_prim.py

```diff
@@ -49,17 +48,14 @@ def __init__(
         load_config=None,
     ):
         # Other values that will be filled in at runtime
-        self._dc = None                         # Dynamics control interface
-        self._handle = None                     # Handle to this articulation
-        self._root_handle = None                # Handle to the root rigid body of this articulation
         self._root_link_name = None             # Name of the root link
-        self._dofs_infos = None
-        self._n_dof = None                      # dof with dynamic control
+        self._n_dof = None
         self._links = None
         self._joints = None
         self._materials = None
         self._visual_only = None
-        self._physics_view = None
+        self._articulation_tree = None
+        self._articulation_view_direct = None
```

Comment:
What does the "direct" imply?

Reply:
Essentially a double-underscore for a really-private member :)